bucket/caffeine.json

```diff
@@ -0,0 +1,15 @@
+{
+    "homepage": "https://www.zhornsoftware.co.uk/caffeine/index.htm",
+    "description": "Prevent PC standby or sleep",
+    "version": "1.64",
+    "license": "Unknown",
```

Comment:
> All the programs are provided as freeware, including free for commercial use.

```suggestion
    "license": "Freeware",
```

Reply:
🙃 `license : freeware`, that's just **keeping it stupid**. 

---

multiple values for "license" or adding a new key("software type") would be much better and if you want a placeholder for situations like these just use "proprietary", that's more clear and relevant, especially since scoop aims to prefer FOSS command line apps.

or I may have missed a better approach to be more clear to users that trust scoop too much.

this may also need to be improved in each manifest. pending some more time to flesh out the important points.

---
oh, 
>All the programs are provided as freeware
is from the second para on homepage https://www.zhornsoftware.co.uk/

---

regards  ratatoeey